lib/charms/opensearch/v0/opensearch_config.py

```diff
@@ -64,6 +64,15 @@ def set_client_auth(self):
             True,
         )
 
+    def set_recover_after_nodes(self, recover_after_nodes: Optional[int] = None):
```

Comment:
Not used anymore, but keeping it as an option.

Reply:
Maybe add a comment here to refer the deprecation notice and - once [your issue](https://github.com/opensearch-project/OpenSearch/issues/15599) has been replied to - remove completely if it is no longer supported.

Reply:
@reneradoi I think even better is to only focus on data nodes tbh. These will stay moving forward.